Documentation/infra-guide/arm64-infra.md

```diff
@@ -66,3 +66,61 @@ If the etcd project needs new `arm64` infrastructure we can open an issue with t
 Note: `arm64` compute capacity is not currently available in all regions, this can be checked with [metal-cli](https://github.com/equinix/metal-cli) `metal capacity get | grep arm`.
 
 [CNCF Community Infrastructure Lab]: https://github.com/cncf/cluster/issues
+
+### Setting up a new github actions runner
+
+Once the new blank machine has been provisioned it needs to be set up as a github actions runner to be able to accept etcd workflow jobs. Follow the steps below to complete this:
+
+1. **Install pre-requisites**
+
+With etcd jobs running inside containers we need to ensure the `docker` container engine is present on the machine.
+
+```bash
+# Ensure all packages are up to date
+sudo apt update && sudo apt upgrade
+
+# Install pre-requisites
+sudo apt install --yes build-essential git wget curl docker.io
+
+# Check the docker service is now started and enabled
+sudo systemctl status docker.service && sudo docker ps
+```
```

Comment:
Can we link to docker official guide? Of course, it's nice to provide verified commands.

https://docs.docker.com/desktop/install/ubuntu/

Reply:
Note those instructions are for docker desktop not docker engine but yeah I get the point.

Currently we intentionally deviate from the official instructions to rely on the `docker.io` package maintained by Ubuntu Team which is a simpler approach than setting up the custom apt repos for official docker packages.

There isn't really much downside to this from what I can see, we run an older but stable release (which still gets updates) last release was late last year.

Welcome feedback from this, if folks want to change the servers from what they have currently I can get it done. Overall I'm pretty neutral with maybe a slight preference to leave things as they are currently.

Reply:
Sure thing, done! 🙂